packages/deployments/contracts/contracts/core/connext/helpers/Executor.sol

```diff
@@ -26,12 +28,35 @@ contract Executor is IExecutor {
 
   // ============ Properties =============
 
+  /**
+   * @notice Address of associated connext
+   */
   address private immutable connext;
+
+  /**
+   * @notice Properties set for access via reentrancy
+   * @dev Contracts that are being called during `execute` may set
+   * properties. Generally, these properties will only be non-null
+   * if the data was authenticated (i.e. reconciled / fraud window
+   * elapsed).
+   *
+   * Callers can access these by using code:
+   * `IExecutor(msg.sender).originSender();`
+   */
   bytes private properties = LibCrossDomainProperty.EMPTY_BYTES;
+
+  /**
+   * @notice Amount transferred
+   * @dev This is always accessible via reentrancy. May *NOT* be the exact amount
```

Comment:
This description feels misleading. This value *should* be accessed via reentrancy, and will be set only for the duration of the execution

Reply:
Let's move this description to all be under that general "Properties" comment under `// ============ Properties =============`